src/app/app-routing.module.ts

```diff
@@ -37,11 +42,11 @@ const routes: Routes = [
     component: UserComponent,
     children: [
       { path: 'your-data', component: YourDataComponent },
-      //   { path: 'your-auctions', component: YourAuctionsComponent },
-      //   { path: 'archival-auctions', component: ArchivalAuctionsComponent },
-      //   { path: 'followed-auctions', component: FollowedAuctionsComponent },
-      //   { path: 'previous-purchases', component: PreviousPurchasesComponent },
-      //   { path: 'messages', component: MessagesComponent },
+      { path: 'your-auctions', component: YourAuctionsComponent },
+      { path: 'archival-auctions', component: ArchivalAuctionsComponent },
+      { path: 'followed-auctions', component: FollowedAuctionsComponent },
+      { path: 'previous-purchases', component: PreviousPurchasesComponent },
+      { path: 'messages', component: MessagesComponent },
```

Comment:
Dont know why these components refuse to be shown on user routing outlet. If you have a minute, check it and let me know what was wrong pls